packages/react-components/theme-designer/src/components/Content/Content.tsx

```diff
@@ -21,14 +17,16 @@ const useStyles = makeStyles({
     alignItems: 'left',
     justifyContent: 'center',
     flexDirection: 'column',
-    ...shorthands.padding('40px', '5%', '0px', '5%'),
+    ...shorthands.padding('40px', '5%', '40px', '5%'),
```

Comment:
_nit:_ You can just do:

```suggestion
    ...shorthands.padding('40px', '5%'),
```